src/Vehicle/Vehicle.cc

```diff
@@ -2638,6 +2645,12 @@ QString Vehicle::_getVehicleFactFileName(MAV_TYPE vehicleType)
     }
 }
 
+void Vehicle::_updateClock(void)
+{
+    _clockFactGroup.currentTime()->setRawValue(QDateTime::currentDateTime().toString(QLocale::system().timeFormat(QLocale::ShortFormat)));
```

Comment:
Please use LongFormat so we get seconds.

Reply:
Using LongFormat makes the string too long for my QGC setup, causing it to overrun its proscribed area.  Because the date format is tied to the computer's date format, a different method of changing the date format would need to be built in to QGC to keep the default setting readable.

![screen shot 2017-10-24 at 1 14 53 pm](https://user-images.githubusercontent.com/10507633/31965823-60fb487e-b8bd-11e7-9872-1bc91eb2ba7d.png)